components/ui/Feed/FeedOverflowButton.tsx

```diff
@@ -0,0 +1,104 @@
+import { IconDots } from "tabler-icons-react-native";
```

Comment:
Given we have a centralized set of APis and state management, I figured we could just create our own `FeedOverflowButton` for the `FeedView`. This just moves the functionality that we had in the FeedView for the overflow menu(s) into its own component, making it easier to maintain and - hopefully in due time - test.